docs/providers/provider-protocol-reference.md

```diff
@@ -4,183 +4,195 @@ category: 63e0c7cf7166bf006d3b4117
 slug: provider-protocol-reference
 ---
 
-All functions described here are bindings generated with fp-bindgen.  The key/values to use for these payloads are exactly the ones you can find in `fiberplane::models` [crate documentation](https://docs.rs/fiberplane-models/latest/fiberplane-models)
+All functions described here are bindings that are generated from our [provider protocol](https://github.com/fiberplane/fiberplane/tree/main/fiberplane-provider-protocol) and which are publish as our [provider bindings crate](https://docs.rs/fiberplane-provider-bindings/latest/fiberplane_provider_bindings/). Please refer to the crate for more details on any of the types mentioned below.
 
-## Imported functions
+We use [`fp-bindgen`](https://github.com/fiberplane/fp-bindgen) for generating the bindings themselves.
```

Comment:
Maybe we could use the docs.rs link for that one too?

Reply:
Nah, we use the GitHub one as the project homepage everywhere already.